File: Angular/app/Eleves/Models/ElevesSearchParameters.ts
export class ElevesSearchParameters {

    constructor() {

        this.Id = null;
        this.Nom = null;
        this.Prenom = null;

        this.Email = null;
        this.License = null;
        this.Club = null;

        this.EvenementId = null;
        this.TypologieId = null;

    }


    public Id: number | null;
    public Nom: string | null;
    public Prenom: string | null;
    public Email: string | null;
    public License: string | null;
    public Club: string | null;
    public EvenementId: number | null;
    public TypologieId: number | null;


}

File: WS/Models/IN/ElevesSearchParameters.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WS.Models.IN
{
    public class ElevesSearchParameters
    {


        public Int32? Id { set; get; }
        public String Nom { set; get; }
        public String Prenom { set; get; }
        public String License { set; get; }
        public String Club { set; get; }
        public Int32? EvenementId { set; get; }
        public Int32? TypologieId { set; get; }

        public String Email { set; get; }

        public String Password { set; get; }

    }
}